### ISSUE TITLE
Release Datetime show the current datetime for package details

### ISSUE THREAD

**Issue by strausmann** [terramar-labs/packages#79]
Hi @tyler-sommer,

i have seen, that the release date is not correct. It is the current datetime and not the releasedate from the package.

![image](https://user-images.githubusercontent.com/4519228/35050189-84801400-fba2-11e7-8bfc-68d869aadab7.png)

![image](https://user-images.githubusercontent.com/4519228/35050231-9b0ffc76-fba2-11e7-8bf1-ae343a808c75.png)

Regards,

Bjoern

**Comment by tyler-sommer** [comment]
Thanks for the bug report!

**Comment by pmacinec** [comment]
The only problem with release datetime is in views, where `release_date` variable is called instead of `releaseDate`, that is declared in `Composer\Package\Package.php`.

Change code on line 34 in `views/Plugin/Satis/Inventory/view.html.twig` from:
```
{{ package.release_date|date('Y-m-d H:i:s') }}
```
to:
```
{{ package.releaseDate|date('Y-m-d H:i:s') }}
```

Have a nice day. :)

**Comment by tyler-sommer** [comment]
Fixed in #106
